### PR TITLE
Review and edits to Protect a web service by using bearer token authentication tutorial 

### PR DESCRIPTION
**Pull request by sheilamjones**
This PR contains minor doc edits, Vale fixes, and final publishing checks to the _Protect a web service by using bearer token authentication_ tutorial guide. 

Reference: QDOCS-538

### CODE REVIEW

**Comment by MichalMaler** [comment]
@sheilamjones Line 37 - Switch to active voice:

A Keycloak server issues the bearer token and represents the subject for which the token was issued.

File: docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc
Comment (MichalMaler) [inline docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc:265]:
```suggestion
 * Log out and log back in as `admin` (password: `admin`), with both `admin` and `user` roles.
```

Comment (sheilamjones) [reply docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc:265]: @MichalMaler: Many thanks for your suggestions. I implemented your earlier comments, however, for this one, I would prefer to keep as it is because similar language to describe the access privileges of other users is used i.e. the users "have" certain roles. I hope this is okay.

File: docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc:259]:
```suggestion
Then, in an `OpenID Connect` card, click the `Keycloak provider` link .
```

Suggesting this minor update because Dev UI contains the cards such as `OpenId Connect`

File: docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc:265]: IMHO `with` does not quite work, one does not login with some roles, someone logs in, and the system checks what roles this user has

Comment (sheilamjones) [reply docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc:265]: I like your suggestion. Updated to "log out and log in again"

**Comment by sheilamjones** [comment]
Hi @sberyozkin, Hi @MichalMaler,

Many thanks for your reviews of this guide. If all looks good from your perspective,  I would appreciate your approvals and then this one is ready to merge.
Kind regards,
Sheila

**Comment by sheilamjones** [comment]
Hi @sberyozkin,
If you get a moment, this PR is ready to merge. 
Many thanks,
Sheila

**Review by MichalMaler — APPROVED** [review]

**Comment by sheilamjones** [comment]
@sberyozkin, this PR is ready to merge when you get a moment. Many thanks.

**Comment by sheilamjones** [comment]
Update: setting back to Draft pending final SME approval.

**Comment by sberyozkin** [comment]
@sheilamjones Can you please check if this PR resolves #38172 ?

**Comment by sheilamjones** [comment]
Hi @sberyozkin,
Rolfe confirmed that the issue outlined in https://github.com/quarkusio/quarkus/issues/38172 does NOT occur on Main. It is only occurring on version 3.6. Therefore, we propose to cherrypick and make the fix directly in the 3.6 branch. Therefore, this PR does not fix that issue.
Kind regards,
Sheila

File: docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc:55]: The numbering scheme is unusual, I know some other OIDC and security tutorials still use it, but these docs are in minority, simple TOC works fine, as far as I recall I removed it based on the feedback from QE, I'd rather keep removing the numbers in other docs instead

File: docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc:194]:
```suggestion
* `quarkus.oidc.client-id` sets a client id that identifies the application.
```

File: docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc:55]: @sheilamjones FYI, the PR which goes to 3.6 will also remove the numbering scheme, so IMHO it is ok without reintroducing it

Comment (sheilamjones) [reply docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc:55]: Hi @sberyozkin, I saw that alright that the numbering was removed in 3.6. I think it was previously removed (or after feedback from QE) because there was an issue with the numbering itself. It wasn't rendering correctly and was rendering with a dot before each number as well as after, that is,  ".1", ".2" etc. 
We fixed this in this latest version on Main, by removing the extra subheading "Procedure".  IMHO, I think the numbering does help to differentiate each individual step of the tutorial. Perhaps, I could remove for now and revisit it again in the future.

**Comment by sheilamjones** [comment]
Thanks @sberyozkin. I made those final corrections. This is ready for merging if all looks okay to you. Many thanks for your time and detailed reviews with these topics. :)

File: docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc]: @sheilamjones OK, sure, lets have a wider discussion, I kind of like a non-numbering scheme, but appreciate you like the alternative, so I will support the numbering one if we have a wide enough consensus :-)

**Review by sberyozkin — APPROVED** [review]
Thanks Sheila

File: docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc
Comment (sheilamjones) [reply docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc]: Sounds good. I removed the numbering from here for now, but think it would be good to revisit this collectively and have a wider consensus on numbering in the tutorials. Many thanks.

**Pull request merged**
